pkg/sloop/ingress/kubewatcher.go

```diff
@@ -310,6 +310,19 @@ func (i *kubeWatcherImpl) processUpdate(kind string, obj interface{}, watchResul
 		// We are only grabbing namespace here for a prometheus metric, so if metadata extract fails we just log and continue
 		glog.V(2).Infof("No namespace for resource: %v", err)
 	}
+	if enableGranularmetrics && kind == "Event" {
+		EventInfo, err1 := kubeextractor.ExtractEventInfo(resourceJson)
```

Comment:
please use lowercase, the uppercase makes it global variables

Reply:
eventInfo

Reply:
You mean eventInfo instead of EventInfo?

Reply:
I was not aware of it. Thanks for letting me know, will change

Reply:
Made the change